pkg/index.yaml

```diff
@@ -300,9 +300,22 @@ entries:
     urls:
     - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/nginx-0.4.2.tgz
     version: 0.4.2
+  rundeck:
+  - apiVersion: v1
+    appVersion: "1.0"
+    created: 2018-06-20T10:48:05.24483179Z
```

Comment:
MUST: Regarding the upstream pr, should go into incubator and not stable.  They will most def require you to do this.